scripts/Tools/case.submit

```diff
@@ -38,9 +38,12 @@ OR
                         help="Do not submit jobs to batch system, run locally.")
 
     parser.add_argument("--prereq",
-                        help="Specify a prerequiset job id, this job will not start until the "
+                        help="Specify a prerequisite job id, this job will not start until the "
                         "job with this id is completed (batch mode only)")
 
+    parser.add_argument("--prereq-allow-failure", action="store_true",
```

Comment:
there should be an additional option here --resubmit-in-queue or --resumbit-immediate to indicate that all jobs should be submitted at once with prereqs